src/form/Form.d.ts

```diff
@@ -0,0 +1,118 @@
+/// <reference types="react" />
+import { FieldValues, setValues } from '../utils/init';
+
+export interface ElementProps {
+  [fieldId: string]: {
+    [propName: string]: string;
+  };
+}
+
+type SetStep = (stepKey: string) => void;
+// am I missing trigger actions?
+type TriggerAction =
```

Comment:
the onChange trigger can only be field or addressSelect currently while the onSubmit and onSkip can be click or change. this is all in our documentation https://docs.feathery.io/develop/react/onsubmit